tests/devices/unit_tests/test_undulator_dcm.py

```diff
@@ -10,11 +10,14 @@
 )
 
 from dodal.devices.dcm import DCM
-from dodal.devices.undulator import Undulator, UndulatorGapAccess
+from dodal.devices.undulator import (
+    Undulator,
+    UndulatorGapAccess,
+    _get_closest_gap_for_energy,
+)
```

Comment:
Should: I don't think it makes sense for the `test_undulator_dcm.py` to be testing the `undulator` itself. This logic should be tested in `test_undulator.py`

Reply:
done